edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/TwinManagerTest.cs

```diff
@@ -1511,39 +1511,58 @@ public async void ConnectionReestablishedDoesNotSyncReportedPropertiesWhenEmptyS
         }
 
         [Fact]
-        public void ValidateTwinPropertiesSuccess()
+        public void ValidateTwinPropertiesWithLongName()
```

Comment:
Do we have a test case that models the reported issue?

Reply:
yes, it is covered by ValidateTwinPropertiesWithLongValue; and the assertion checks the message is complaint to the expected property, not from the json root.